### ISSUE TITLE
[PostgreSQL] [Pro] Triggers are not being persisted to PostgreSQL

### ISSUE THREAD

**Issue by liquibot** [liquibase/liquibase#1135]
## Environment
**Liquibase Version**: 3.8.9

**Liquibase Integration & Version**: CLI

**Liquibase Extension(s) & Version**: none

**Database Vendor & Version**: PostgreSQL 12

**Operating System Type & Version**: Docker

## Description
Triggers not created in PG 12

## Steps To Reproduce
Using the Sakila schema, I created the schema, generatedchangelog, and pushed to empty database. Even though I had the following in my changelog, the Trigger was not created:

```
    <changeSet author="r2 (generated)" id="1588785396440-93">
        <pro:createTrigger disabled="false" path="objects/trigger/last_updated.sql" relativeToChangelogFile="true" tableName="film" triggerName="last_updated"/>
    </changeSet>

```

## Actual Behavior
_confused travolta_
Where are my triggers?

## Expected/Desired Behavior
triggers created

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-245) by [Unito](https://www.unito.io/learn-more)
┆Fix Versions: Liquibase 3.10.3

**Comment by molivasdat** [comment]
Hi @r2datical .  Thank you for writing up this issue.  We will add it to our list of issues to process.

**Comment by molivasdat** [comment]
Closing as closed in 3.10.3 and 4.1
